cities/prague_pyvo.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime
+
+from calendar_data.data import parse_calendar_data
+from telegram_bot.bot import send_message
+
+
+def send_message_to_prague_channel():
+    events = parse_calendar_data("https://pyvo.cz/api/series/praha-pyvo.ics")
+    prague_chat_id = "-1001837942773"  # test chat
+    for event in events:
+        summary = event.get("summary").replace("(", "").replace(")", "")
+        event_date = event.get("dtstart").dt.replace(tzinfo=None).date()
+        output_event_date = event.get("dtstart").dt.date().strftime("%d.%m.%Y")
+        date_difference = abs(event_date - datetime.today().date()).days
```

Comment:
the `abs` is not needed here since `parse_calendar_data` only includes future events and can be confusing. If it didn't, it would also be wrong as it would alert on events 3/7 days in the past.